API/01_API.csproj

```diff
@@ -41,18 +41,17 @@
         <PackageReference Include="Microsoft.OpenApi" Version="1.1.4" />
         <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="3.1.1" />
         <PackageReference Include="NetEscapades.Configuration.Validation" Version="2.0.0" />
-        <PackageReference Include="RestSharp" Version="106.11.2" />
+        <PackageReference Include="RestSharp" Version="106.11.4" />
         <PackageReference Include="Sentry.AspNetCore" Version="2.1.1" />
         <PackageReference Include="Sentry.Serilog" Version="2.1.1" />
         <PackageReference Include="Swashbuckle.AspNetCore" Version="5.0.0" />
-        <PackageReference Include="Serilog.AspNetCore" Version="3.2.0" />
+        <PackageReference Include="Serilog.AspNetCore" Version="3.4.0" />
     </ItemGroup>
 
     <ItemGroup>
         <ProjectReference Include="..\Data\06_Data.csproj" />
         <ProjectReference Include="..\MessageBrokerPublisher\09_MessageBrokerPublisher.csproj" />
         <ProjectReference Include="..\Models\07_Models.csproj" />
-        <ProjectReference Include="..\NotificationSystem\11_NotificationSystem.csproj" />
```

Comment:
This dependency snuck in from some reason, basically the API should only depend on the MessageBrokerPublisher, this because we want the notificationsystem to be decoupled from the API. So the API should publish a task to the message broker, then the notification system gets notified by the messagebroker and does its job. 
This way we could deploy the notificationsystem if necessary on a different server.